pinot-query-planner/src/main/java/org/apache/pinot/query/planner/serde/ProtoSerializationUtils.java

```diff
@@ -148,7 +148,7 @@ private static Plan.MemberVariableField serializeMemberVariable(Object fieldObje
     } else if (fieldObject instanceof String) {
       builder.setLiteralField(stringField((String) fieldObject));
     } else if (fieldObject instanceof byte[]) {
-      builder.setLiteralField(bytesField(ByteString.copyFrom((byte[]) fieldObject)));
+      builder.setLiteralField(bytesField((byte[]) fieldObject));
```

Comment:
this is still asymmetric. we should either
1. do not allow `instanceof byte[]` here 
2. convert back to byte[] intead of ByteArray

i knew that some classes are all converted into bytes and cannot be converted back to their original format, but we should explicitly structure those --> if this will occur many special fields in the future, i propose:

encode the bytesField, with type enum and then the blob. for example
```
enum ProtoEncodedByteType {
byte[] --> 0 
ByteArray --> 1
ByteString --> 2
BigDecimal --> 3
GeogrianCalendar --> 4
...
}
```
and have a ser/de util specific for bytes field

Reply:
ignored my comment here. we have separate field in proto going forward. so this is solely for byte[]